### ISSUE TITLE
Force-remove packages that stopped marking stable version from stable.melpa.org

### ISSUE THREAD

**Issue by doublep** [melpa/melpa#8587]
I'll go with an example, but this is _by far_ not the only case. Package Smartparens at stable.melpa.org is at version 1.11.0. If we look at the [source code](https://github.com/Fuco1/smartparens/tree/1.11.0), this version is *six years old*. However, [current "unstable" version](https://github.com/Fuco1/smartparens/tree/master) is 2 weeks old and 230 commits after that.

Now let's consider any user who prefers stable packages. For whatever reason. I mean, as long as you have stable.melpa.org around, you sort of assume that there are such people. By default, this user would likely install every new package in a stable version — because, why would they even check? If there are no stable versions at all, Emacs package system would fall back to using melpa.org, which is fine: not like there is any other option. But in case of Smartparens or something similar Emacs will _silently_ install the hopelessly obsolete version, without _any_ indication that there might be something fishy here.

In effect, by marking some stable versions years ago and then just dropping this, such packages make stable.melpa.org largely unusable. Either you have to single-out those few packages you know you want in stable version, or _every time_ you want to try something new, you need to go out and check: "Is this stable version even up-to-date or did they just mark it and then decided: 'meh, we never do it again' ten years ago?"

Proposal: go through the list of packages and find those that have no stable versions in over a year and >30 commits since the last version. Send the maintainers a mail that either they should mark a stable version or get removed from stable.melpa.org. I'd automate this if possible.

Advantages: stable.melpa.org becomes usable for those few who use it: you know if the package is there, then its stable versions are more or less up-to-date. Disadvantages: not that I see any, those packages are still available from melpa.org and obviously don't care about stable.melpa.org to begin with.

**Comment by milkypostman** [comment]
I agree we should get rid of stable :)

**Comment by minad** [comment]
@tarsius I assume this issue can be closed too, given the new releases/snapshots channels.

**Comment by tarsius** [comment]
Yeah, we won't remove the stable/releases channels and we won't remove individual packages from these channels (for this reason, that is).

The alternative next step will be that I publish some statistics (so people can assess the severity of the situation) and lists of problematic packages. When it comes to individual packages, the new version string format (`RELEASE.0.DATE.COUNT`) allows a quick initial assessment. If this indicates that there might be a problem, a user will have to consult the git history, to decide whether they feel comfortable using the package anyway. We cannot make that decision for them.

Not every package that hasn't been released in a long time is in the same situation. Simply removing all beyond an arbitrary threshold would definitely be wrong. The alternative is to manually assess each package individually. Nobody is going to volunteer to do that work, so this won't ever happen.
